src/visitpy/common/visitmodule.C

```diff
@@ -10062,13 +10062,20 @@ GetNamesHelper(PyObject *self, PyObject *args, stringVector &names)
 {
     PyObject *tuple;
     char *str;
+    int i;
     bool retval = false;
 
     if(PyArg_ParseTuple(args, "s", &str))
     {
         names.push_back(str);
         retval = true;
```

Comment:
Should there be a `PyErr_Clear();` here?

Reply:
In this if condition `PyErr_Clear();` is not actually needed, since no error messages are thrown here.

For the other cases the error mentioned above:

> ```shell
> TypeError: argument 1 must be str, not int
> ```

would occur. `PyErr_Clear();` removes this error.

Reply:
Got it! Thanks for the explanation.